### ISSUE TITLE
Add a Projects/Tools page

### ISSUE THREAD

**Issue by iaine** [codemeta/codemeta#98]
Add a Projects/Tools page to link to existing projects to help find existing tools and projects. Issued as pull request #97

**Comment by iaine** [comment]
I was thinking some along the lines of the ZeroMQ project labs  [http://zeromq.org/docs:labs] with its links. The CodeMeta project would not host but would provide a template for adding a project (and perhaps some rough categories to organise them : file generation, integration and so on)?

**Comment by moranegg** [comment]
I'm very interested in the tools page, I found a version in the #97 pull request, but I can't find it in the latest revision.
specifically, I'm searching for tools translating/parsing package.json and pom.xml files to codemeta, preferably without using the entire package for generating the codemeta.json.

**Comment by cboettig** [comment]
@moranegg we're really pretty early in the tools development side.  Some of the early tools still need a little updating to use the latest version of the codemeta context.

- @arfon's Ruby-based generator: https://gist.github.com/arfon/478b2ed49e11f984d6fb 
- @mfenner's Ruby-based tools for DataCite, including translating codemeta: https://github.com/datacite/bolognese
- The `codemetar` R package, which has some examples for how to translate from other types such as package.json into codemeta here: https://codemeta.github.io/codemetar/articles/translating.html

Happy to work with you on the `codemetar` side to streamline this, but for any standard that isn't already in JSON-LD there's going to be some additional coding necessary to parse the original format (though with `package.json` this will be very easy).  

There's a bunch of tooling development on the 'back end' for databases, (arguably `bolognese` is an example of that) in which there's tooling for specific databases working with codemeta, but may still be useful to feature on the tools page (e.g. work from DataONE, Zenodo, and later from figshare)

**Comment by moranegg** [comment]
Thanks @cboettig the article is very useful, I was sure that CodemetaR was only for generating the codemeta.json file from an R package and i'm really happy to see other functionalities are available. If I have more questions about CodemetaR I will open issues there.

I'm trying an indexing procedure for the 'back end' of our SoftwareHeritage database where I need to find manifest files and package management files, extract the metadata and translate it. Recently we have decided to use the codemeta vocabulary for the translation mainly  because of the crosswalk table.

Now I'm searching for available dedicated tools for metadata translation. We are using pygments for language detection and nomossa for license detection.

[bolognese](https://github.com/datacite/bolognese) it is great but useful for citation purposes (so no package management support, as I understood it)

I also found [bibliothecary](https://github.com/librariesio/bibliothecary) that parses dependencies from the manifest files and I don't know if libraries.io has a tool translating to codemeta the metadata found.
